src/acq.c

```diff
@@ -44,11 +44,11 @@ void acq_set_prn(u8 prn)
  * \param cp  Code phase of best point.
  * \param cf  Carrier frequency of best point.
  */
-void acq_send_result(u8 prn, float snr, float cp, float cf)
+void acq_send_result(u16 prn, float snr, float cp, float cf)
 {
   msg_acq_result_t acq_result_msg;
 
-  acq_result_msg.sid = prn; /* TODO prn -> sid */
+  acq_result_msg.sid.prn = prn; /* TODO prn -> sid */
```

Comment:
Clear the TODO? Or is it still helpful to leave it around to use the other information in the structure?